tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -1227,6 +1230,13 @@ class DxcCompiler : public IDxcCompiler3,
       CComPtr<IDxcResult> pResult;
       hr = e.hr;
       std::string msg("Internal Compiler error: ");
+      switch (hr) {
+      case DXC_E_VALIDATOR_MISSING:
+        msg = "Error: external validator selected, but DXIL.dll not found.";
```

Comment:
This doesn't seem to be tested. I know @pow2clk was working on related things. Is this intended to be covered by the work you have going on?

Reply:
It is. I couldn't add the testing just yet because this is in staging and my PR is in main.

Reply:
Issue created: https://github.com/microsoft/DirectXShaderCompiler/issues/5991